packages/fhir-converter/src/templates/cda/DataType/Quantity.hbs

```diff
@@ -0,0 +1,42 @@
+{{#with (decodeHtmlEntities quantity.value) as |decodedValue|}}
+  {{#with (convertFeetAndInchesToCm decodedValue) as |feetAndInches|}}
+    {{#if feetAndInches.isValid}}
+      {
+        "value": "{{feetAndInches.value}}",
+        "unit": "cm",
+        "system": "http://unitsofmeasure.org"
+      }
+    {{else}}
+      {{#with (extractComparator decodedValue) as |comparator|}}
+        {{#if comparator.isValid}}
+          {
+            "value": "{{comparator.number}}",
+            "comparator": "{{{comparator.comparator}}}",
```

Comment:
three braces here so that the special char ">=" renders as a special char and not html junk